shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -21,6 +21,10 @@
 // it is activated.
 static constexpr double kUITextInputAccessibilityEnablingDelaySeconds = 0.5;
 
+// A delay before reenabling the UIView areAnimationsEnabled to YES
+// in order for becomeFirstResponder to recieve the proper value
```

Comment:
```suggestion
// in order for becomeFirstResponder to receive the proper value
```

Reply:
Thank you! Those will slip by me from time to time.